src/styles/quote.css

```diff
@@ -0,0 +1,84 @@
+.quotes-container {
+  position: relative;
+  transform: none;
+  left: 0;
+  max-width: 76em;
+  margin: auto;
+}
+
+@media only screen and (max-width: 720px) {
+  .quotes-container {
+      width: 45em;
+      display: flex;
+      justify-content: center;
+      flex-direction: column;
+      text-align: center;
+      justify-content: center;
+      position: relative;
+      margin-top: 190px;
+      margin-left: 165px;
+  }
+}
+
+@media only screen and (max-width: 485px) {
+  .quotes-container {
+      width: 45em;
+      display: flex;
+      justify-content: center;
+      flex-direction: column;
+      text-align: center;
+      justify-content: center;
+      position: relative;
+      margin-top: 130px;
+      margin-left: 50px;
+  }
+}
+
+.customer-blockquote {
+  padding: 24px;
+  width: calc(50% - 55px);
```

Comment:
Please add spaces between opening and closing parentheses, and arguments, in a function invocation or control flow expression:
```js
fn(param1, param2) { /* ... */ }
if(expression) { /* ... */ }
```
should be
```js
fn( params1, param2 ) { /* ... */ }
if( expression ) { /* ... */ }
```